deploy/kiali/kiali_cr.yaml

```diff
@@ -431,10 +431,12 @@ spec:
 # **Istio-specific settings:
 # istio_identity_domain: The annotation used by Istio to identify domains.
 # istio_sidecar_annotation: The pod annotation used by Istio to identify the sidecar.
+# istio_status_enabled: Enable/Disable of istio component status into masthead indicator. It defaults to true.
 # url_service_version: The Istio service used to determine the Istio version.
 #                      If empty, assumes it is in the istio namespace at the URL "http://istio-pilot.<istio_namespace>:8080/version"
 #    ---
 #    istio:
+#      istio_status_enabled: true
```

Comment:
keep this list alphabetized - move it down to line 441 (after istio_sidecar_annotaiton).